platform/metrics/platform-metrics-reporting/src/main/java/ddf/metrics/reporting/internal/rest/MetricsEndpoint.java

```diff
@@ -253,7 +253,7 @@ public Response getMetricsData(
         ByteArrayInputStream bis = new ByteArrayInputStream(metricsGraphBytes);
         response = Response.ok(bis, PNG_MIME_TYPE).build();
       } catch (IOException | MetricsGraphException e) {
-        LOGGER.info("Could not create graph for metric {}", metricName);
+        LOGGER.info("Could not create graph for specified metric");
```

Comment:
Just curious what the motivation behind these changes were

Reply:
In this case, log forgery. It's possible to hit that endpoint in a way that presents a fairly legitimate looking log to the user if they're simply looking at `log:tail` or the `ddf.log`. Sure, it's not in `security.log`, but since the metricName arguably isn't adding much benefit anyway, we took it out.

Reply:
I think this case is fine, but I wouldn't go overboard cleaning up log forgery findings anywhere outside of the audit log. It's of minimal risk and provides minimal value.

Reply:
Yeah, others that were found typically required such convoluted execution paths and/or stars to align that because they wouldn't even "pay off" with an audit in the audit log, I didn't bother dealing with them.

Reply:
Scott configured the log to strip out newlines from log messages. Forgers need newlines to make a log forgery look convincing. If newlines are not stripped:

```
| legit date  | legit component  | legit message
| forged date | forged component | forged message
```

Once you remove newlines, the forgery is no longer disguised:
 ```
| legit date | legit component | legit message | forged date | forged date| forged message
```

It doesn't stop bad data from getting into the log, but it does much to mitigate the vulnerability.